Godot_test/ClientScripts/EquipmentItem.cs

```diff
@@ -0,0 +1,40 @@
+using Godot;
+using Roguelike.Tiles.Looting;
+using System;
+
+public partial class EquipmentItem : Node
+{
+	public Roguelike.Tiles.Looting.Equipment equipment;
+  private string pathToEquipment = "res://Sprites/Loot/Equipment/";
+
+  public override void _Ready()
+  {
+		base._Ready();
+		CallDeferred("SetSprite");
+  }
+
+  public void SetSprite()
+	{
+		var spr = (Sprite2D)GetChild(0);
+		if (equipment is Weapon w)
+		{
+	  var texture = pathToEquipment + "Weapons/";
+	  if (w.kind == Weapon.WeaponKind.Sword)
```

Comment:
Not obligatory, but as exercise we could have Dictionary<Weapon.WeaponKind, string> weapnKindPaths = {Weapon.WeaponKind.Sword, "Sword"}  ...

Then we could not have if's at all.